front/app/containers/IdeasNewPage/IdeasNewIdeationForm/index.tsx

```diff
@@ -148,13 +148,10 @@ const IdeasNewIdeationForm = ({ project }: Props) => {
   };
 
   const onSubmit = async (data: FormValues) => {
-    let location_point_geojson;
-
-    if (data.location_description && !data.location_point_geojson) {
-      location_point_geojson = await geocode(data.location_description);
-    } else {
-      location_point_geojson = await getLocationGeojson(initialFormData, data);
-    }
+    const location_point_geojson = await getLocationGeojson(
+      initialFormData,
+      data
+    );
```

Comment:
This util function already handles all the necessary cases to make sure the proper `location_point_geojson` is being saved. Additionally, using only the util function solves the current bug where the coordinates in the URL aren't being saved correctly.